homeassistant/components/slide_local/button.py

```diff
@@ -0,0 +1,57 @@
+"""Support for Slide button."""
+
+from __future__ import annotations
+
+import logging
+
+from homeassistant.components.button import ButtonDeviceClass, ButtonEntity
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import SlideConfigEntry
+from .coordinator import SlideCoordinator
+from .entity import SlideEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+PARALLEL_UPDATES = 0
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: SlideConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up button for Slide platform."""
+
+    coordinator = entry.runtime_data
+
+    async_add_entities(
+        [
+            SlideButton(
+                coordinator,
+            )
+        ]
+    )
+
+
+class SlideButton(SlideEntity, ButtonEntity):
+    """Defines a Slide button."""
+
+    _attr_entity_category = EntityCategory.CONFIG
+    _attr_translation_key = "calibrate"
+
+    def __init__(
+        self,
+        coordinator: SlideCoordinator,
+    ) -> None:
+        """Initialize the slide button."""
+        super().__init__(coordinator)
+
+        self._attr_device_class = ButtonDeviceClass.IDENTIFY
```

Comment:
Calibrate is not identifying a device per se. Identifying is use for example for network devices where the light starts blinking or a light in a network that start blinking

Reply:
Ok, makes sense. So there is no DeviceClass that would suit?

Reply:
Yep, no device class